src/installer/tests/TestUtils/Command.cs

```diff
@@ -196,7 +197,22 @@ public Command Start()
 
             ReportExecBegin();
 
-            Process.Start();
+            // Retry if we hit ETXTBSY due to Linux race
+            // https://github.com/dotnet/runtime/issues/58964
+            for (int i = 0; i < 3; i++)
+            {
+                try
+                {
+                    Process.Start();
+                    break;
+                }
+                catch (Win32Exception e) when (e.Message.Contains("Text file busy"))
+                {
+                    // 10 ms is short, but the race we're trying to avoid is in-between
+                    // "fork" and "exec", so it should be fast
+                    Thread.Sleep(10);
```

Comment:
Ideally this should print out some diagnostics - so that we know it happened. Not sure if Console.WriteLine is the correct solution here - but it might be.

Reply:
Since this has been giving us trouble, having enough info to diagnose potential issues sounds like a good idea.
I know that we don't have existing diagnostics, but ideally that should not block us from adding them :wink:.

That said - I don't know how the diagnostics should look. Is there some existing mechanism for tests to output additional info to be used in case of failure (but honestly it would be great to get this even on passes if it's not too verbose).

Reply:
Unfortunately I don't know of a good logging mechanism at the moment. This code could be executed across many threads, so there would likely need to be some coordination for anything useful to come out.